File: 1009.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main() {
	
	string nome;
	double salarioFixo, montante, total;
	
	cin >> nome;
	cin >> salarioFixo;
	cin >> montante;
	
	total = (0.15*  montante) + salarioFixo;
	
	cout << "TOTAL = R$ " <<  fixed << setprecision(2) << total << endl;
	
	return 0;
}	
	

File: JavaScript/1009.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var nome = lines.shift()
var salarioFixo = parseFloat(lines.shift());
var montante = parseFloat(lines.shift());

total = ((montante * 15) / 100) + salarioFixo

console.log('TOTAL = R$' + total.toFixed(2))